nanome_rmsd/RMSD.py

```diff
@@ -19,6 +19,8 @@ def start(self):
         self._menu = RMSDMenu(self)
         self._menu.build_menu()
         self.selected_before = []
+        self._mobile = []
+        self._target = None
```

Comment:
you should add the self.compare_index here so that it is never undeclared.
Additionally you could provide a comment to explain its purpose.